packages/ckeditor5-ui/src/dialog/dialog.ts

```diff
@@ -189,6 +190,7 @@ export default class Dialog extends Plugin {
 		view.setupParts( {
 			icon,
 			title,
+			hasCloseButton,
```

Comment:
I think we're missing a unit test that verifies that this property gets passed down from `show()` to `setupParts()`. Somewhere in or around `it( 'should setup the view parts with the passed arguments', () => {`?